utils/releaser/src/ReleaseWorker/ReleaseCandidate/TestYourBranchLocallyReleaseWorker.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\Releaser\ReleaseWorker\ReleaseCandidate;
+
+use PharIo\Version\Version;
+use Shopsys\Releaser\ReleaseWorker\AbstractShopsysReleaseWorker;
+use Shopsys\Releaser\Stage;
+use Symplify\MonorepoBuilder\Release\Message;
+
+final class TestYourBranchLocallyReleaseWorker extends AbstractShopsysReleaseWorker
+{
+    /**
+     * @param \PharIo\Version\Version $version
+     * @return string
+     */
+    public function getDescription(Version $version): string
+    {
+        return 'Test your branch locally - running composer-dev, standards and tests - this might take a few minutes"';
+    }
+
+    /**
+     * Higher first
+     * @return int
+     */
+    public function getPriority(): int
+    {
+        return 750;
+    }
+
+    /**
+     * @param \PharIo\Version\Version $version
+     */
+    public function work(Version $version): void
+    {
+        try {
+            $this->processRunner->run('php phing composer-dev standards tests', true);
+        } catch (\Symfony\Component\Process\Exception\ProcessFailedException $ex) {
+            $this->symfonyStyle->caution($ex->getProcess()->getOutput());
+            $this->symfonyStyle->note('A problem occurred, check the output a fix it please.');
+            $runChecksAgain = $this->symfonyStyle->ask('Run the checks again?', 'yes');
+            if ($runChecksAgain) {
+                $this->work($version);
+            }
+        }
+        $this->symfonyStyle->success(Message::SUCCESS);
```

Comment:
if there will be more problems SUCCESS will be alerted also the same amount of time.

Reply:
well, I would not say...SUCCESS is alerted only in the end - whether when there is no problem or when you do not want to run the checks again...Let us discuss this